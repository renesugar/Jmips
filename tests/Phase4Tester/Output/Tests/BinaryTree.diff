--- SelfTestCases/BinaryTree.out	2019-02-02 14:36:13.000000000 -0800
+++ ./Output/Tests/BinaryTree.out	2019-03-13 22:31:43.000000000 -0700
@@ -1,27 +1,19 @@
-16
-100000000
-8
-16
-4
-8
-12
-14
-16
-20
-24
-28
-1
-1
-1
-0
-1
-4
-8
-14
-16
-20
-24
-28
-0
-0
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 90 column 12: "100000000": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 274 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 293 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 308 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 362 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 403 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 418 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 434 column 16: "$t1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 439 column 16: "$t2": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 440 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 493 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 622 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 650 column 16: "$s1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 688 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 699 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 1084 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 1111 column 16: "$s0": operand is of incorrect type
 
+Processing terminated due to errors.
