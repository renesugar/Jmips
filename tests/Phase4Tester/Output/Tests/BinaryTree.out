Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 90 column 12: "100000000": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 274 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 293 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 308 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 362 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 403 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 418 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 434 column 16: "$t1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 439 column 16: "$t2": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 440 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 493 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 622 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 650 column 16: "$s1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 688 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 699 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 1084 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/BinaryTree.compiler.out line 1111 column 16: "$s0": operand is of incorrect type

Processing terminated due to errors.
